scripts/launch-xrd

```diff
@@ -243,12 +244,18 @@ while [[ $# -gt 0 ]]; do
             shift 2
             ;;
         -- )
-           shift
-           ;;
+            shift
+            if [[ $# -eq 0 ]]; then
+                echo "Invalid syntax: No args after --." >&2
```

Comment:
Suggested rewording, aiming for consistency with other errors. This isn't invalid *syntax*, it's an invalid *invocation*.
```suggestion
                echo "Error: Expected at least one arg after the '--' separator." >&2
```